compiler-plugin/src/main/java/io/ballerina/stdlib/websocket/plugin/Utils.java

```diff
@@ -339,7 +338,7 @@ private static void repoteDiagnostics(String functionName, FunctionDefinitionNod
 
     private static boolean validateReturnType(TypeSymbol symbol) {
```

Comment:
Shall we do a small refactoring to this method to improve the readability ? We can have a list of all the allowed types and check whether `symbol.typeKind()` is present in that list.

Reply:
+1

Reply:
Refactored with [7622da7](https://github.com/ballerina-platform/module-ballerina-websocket/pull/1121/commits/7622da7b90f8ee08b9b9730cd4ac92ece53e23f7)